tests/transfer/test_transfer_key_to_key.py

```diff
@@ -2,7 +2,13 @@
 import re
 from playwright.sync_api import Page, expect
 from vega_sim.service import VegaService
-from actions.utils import wait_for_toast_confirmation
+from actions.utils import wait_for_toast_confirmation, create_and_faucet_wallet, WalletConfig, next_epoch
+import vega_sim.proto.vega as vega_protos
+
+LIQ = WalletConfig("liq", "liq")
+PARTY_A = WalletConfig("party_a", "party_a")
```

Comment:
with the referrals work that will go in after the potential bug, I think the party a/b stuff could potentially be somewhere central. Like a generic keys file with things like this included. Nothing to address here but an observation

Reply:
yeah agreed!